src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.jsx

```diff
@@ -135,6 +135,8 @@ function CustomFieldInputMultipleChoice(props) {
             {!props.readOnly && (
               <input
                 aria-labelledby={`${props.id}-label`}
+                aria-haspopup={'listbox'}
+                aria-expanded={renderPopup}
```

Comment:
Looks like we will need to add `role="combobox"` to the input:

https://www.w3.org/TR/wai-aria/#combobox

It makes sense since we want the voice reader to know that this is a "smart widget"

Reply:
Also, I just followed the linter error:

https://circleci.com/gh/mavenlink/design-system/2331?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/role-supports-aria-props.md
https://www.w3.org/TR/wai-aria/#aria-expanded
https://www.w3.org/TR/wai-aria/#combobox